### DISCUSSION TITLE
USB host for dedicated wifi dongle ; BG/N and AC ( mode ?) until WIFI 6 ?

### DISCUSSION THREAD

**Discussion by JPB1JPB1** [hathach/tinyusb#930] [Q&A]
Hello

any chance to get the usb host wifi support for few dedicated wifi dongle to support B/G/N and ac, or Wifi 6 speed support and better ...
Keep going
best Regards
Jp

**Comment by hathach** [comment]
I guess It involves rnsids, but possible If someone is willing to submit pr for it.
